packages/jaeger-ui-components/src/TraceTimelineViewer/SpanDetail/KeyValuesTable.tsx

```diff
@@ -108,7 +108,7 @@ export default function KeyValuesTable(props: KeyValuesTableProps) {
   const { data, linksGetter } = props;
   const styles = useStyles2(getStyles);
   return (
-    <div className={cx(styles.KeyValueTable)} data-test-id="KeyValueTable">
+    <div className={cx(styles.KeyValueTable)} data-testid="KeyValueTable">
```

Comment:
This selector is also used in `TraceView.test`, so this change breaks it. I think changing usages of `data-test-id` to `data-testid` there should fix it.

Reply:
Thanks, its changed now 🙂